### PR TITLE
Issue#1196 hotfix

### LINKED ISSUE #1196: Update movie view according to data model
Comments are saved here: https://projects.invisionapp.com/d/main#/console/18708905/391485091/preview

### PR DESCRIPTION
**Pull request by Maxstgt**
close #1196

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/movie/view/view.component.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export class MarketplaceMovieViewComponent implements OnInit {
           .some(({ movieIds }) => movieIds.includes(this.movieQuery.getActive().id));
       })
     );
+    this.movie$.subscribe(d => console.log(d))
```
Comment (pldespaigne) [inline apps/catalog/marketplace/marketplace/src/app/movie/view/view.component.ts]: did you forget console.log

File: libs/catalog/src/lib/components/production-informations/production-informations.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<h3>Production information</h3>
+<div fxLayout="row nowrap" fxLayoutAlign="start start" fxLayoutGap="30px">
+    <span fxLayout="column">
```
Comment (Hugo-Hontans) [inline libs/catalog/src/lib/components/production-informations/production-informations.component.html]: I don't like this. You use a `<span>` with `fxLayout="column"`. Not make really sens. Why don't use a div or article ?

File: libs/catalog/src/lib/components/production-informations/production-informations.component.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        </p>
+        <div *ngFor="let company of productionCompanies; let i = index" fxLayout="row nowrap">
+            <ng-container *ngIf="i <= 1; else end">
+                {{company.firstName}}
+                {{company.lastName}}
+            </ng-container>
+        </div>
+        <ng-template #end>...</ng-template>
+    </span>
+    <span fxLayout="column">
```
Comment (Hugo-Hontans) [inline libs/catalog/src/lib/components/production-informations/production-informations.component.html]: Same

File: libs/catalog/src/lib/components/production-informations/production-informations.component.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+@Component({
+  selector: 'catalog-production-informations',
+  templateUrl: './production-informations.component.html',
+  styleUrls: ['./production-informations.component.html']
+})
+export class CatalogProductionInformationsComponent {
+  @Input() productionCompanies;
```
Comment (Hugo-Hontans) [inline libs/catalog/src/lib/components/production-informations/production-informations.component.ts]: Type your inputs

File: libs/catalog/src/lib/components/reserved-territories/reserved-territories.component.ts [hunk 5]
```diff
@@ -15,9 +15,5 @@ import { Component } from '@angular/core';
   }`]
 })
 export class CatalogReservedTerritoriesComponent {
-  public reservedTerritories: string[] = [
-    'France (The Jokers)',
-    'USA (Paramount)',
-    'UK (Avalance)'
-  ];
+  @Input() reservedTerritories;
```
Comment (Hugo-Hontans) [inline libs/catalog/src/lib/components/reserved-territories/reserved-territories.component.ts]: Type your input

File: apps/catalog/marketplace/marketplace/src/app/movie/view/view.component.ts (hunk 1 above)
Comment (Hugo-Hontans) [reply apps/catalog/marketplace/marketplace/src/app/movie/view/view.component.ts]: You forgot it

File: libs/movie/src/lib/movie/components/display-principal-informations/display-principal-informations.component.ts [hunk 6]
```diff
@@ -10,4 +10,6 @@ import { Component, ChangeDetectionStrategy, Input } from "@angular/core";
 export class MovieDisplayPrincipalInformationsComponent {
   @Input() main;
   @Input() salesCast;
+  @Input() budget;
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/components/display-principal-informations/display-principal-informations.component.ts]: You don't type inputs here

**Pull request merged**